app/controllers/returns/credits/ClaimForWhichYearController.scala

```diff
@@ -75,6 +75,7 @@ class ClaimForWhichYearController @Inject()(
               selectedRange => {
                 request.userAnswers
                   .setOrFail(JsPath \ "credit" \ selectedRange.key \ "endDate", selectedRange.to)
+                  .setOrFail(JsPath \ "credit" \ selectedRange.key \ "fromDate", selectedRange.from)
```

Comment:
Add this to the safeDeploy changes, for backwards compatible

Reply:
https://github.com/hmrc/plastic-packaging-tax-returns/pull/165#discussion_r1198773518